### ISSUE TITLE
Release 2.0.2 ?

### ISSUE THREAD

**Issue by diazwatson** [elgentos/magento2-product-duplicate-images-remove#4]
When is the backwards compatibility with older versions of PHP/Magento going out? 😄

**Comment by peterjaap** [comment]
Now!

**Comment by diazwatson** [comment]
Do we need to do anything to update the version in [packagist](https://packagist.org/packages/elgentos/magento2-product-duplicate-images-remove)?

![Screenshot 2024-08-05 at 12 21 32](https://github.com/user-attachments/assets/af17d424-152b-444f-8cad-dcf79b85eaca)

**Comment by peterjaap** [comment]
Yes;

```
Reading composer.json of elgentos/magento2-product-duplicate-images-remove (2.0.2)
Importing tag 2.0.2 (2.0.2.0)
Skipped tag 2.0.2, Invalid package information: 
require.magento/framework : invalid version constraint (Could not parse version constraint >=103.0.*: Invalid version string "103.0.*")
Reading composer.json of elgentos/magento2-product-duplicate-images-remove (2.0.1)
```

**Comment by peterjaap** [comment]
Fixed and issued 2.0.3; https://packagist.org/packages/elgentos/magento2-product-duplicate-images-remove
